### DISCUSSION TITLE
Autologin After Verifying not on account creation

### DISCUSSION THREAD

**Discussion by donnelly-consulting** [janko/rodauth-rails#281] [General]
I've disabled autologin on account creation using  create_account_autologin? false. But i do need autologin after account verification. So in rodauth_main.rb I have

`    after_verify_account do
      autologin_session('verify_account')
    end
` but it doesn't login the user  in - any ideas?

**Comment by janko** [comment]
Autologin behavior after account verification is controlled by the `verify_account_autologin?` configuration method. But the `verify_account` feature already sets this to `true` by default.
